conda/utils.py

```diff
@@ -254,3 +257,30 @@ def sys_prefix_unfollowed():
     except Exception:
         return sys.prefix
     return unfollowed
+
+
+def wrap_subprocess_call(on_win, root_prefix, prefix, command):
+    env = environ.copy()
+    tmp_prefix = abspath(join(prefix, '.tmp'))
+    script_caller = None
+    if on_win:
+        comspec = environ[str('COMSPEC')]
```

Comment:
This environment lookup can fail but perhaps allowing the exception to propagate is the best option here.